recipes-core/images/adlink-image-validation.bb

```diff
@@ -0,0 +1,27 @@
+DESCRIPTION = "Adlink Image to validate Ampere machines,COM HPC - AVA/AP1 Platforms  \
+This image contains everything used to test COM HPC Platform , This creates a very large image, not \
+suitable for production."
+LICENSE = "MIT"
+
+inherit core-image
+
+WKS_FILE:comhpc = "adlink-efidisk.wks"
+
+## Select Image Features
+IMAGE_FEATURES += " \
+    debug-tweaks \
+    tools-profile \
+    nfs-server \
+    tools-debug \
+    ssh-server-dropbear \
+    tools-testapps \
+"
+
+CORE_IMAGE_EXTRA_INSTALL += " \
+    packagegroup-core-full-cmdline \
+    packagegroup-core-base-utils \
+"
+
+IMAGE_INSTALL:append = " usbutils i2c-tools libgpiod libgpiod-tools bmap-tools iperf3 ipmitool pciutils brcmswitch"
```

Comment:
usbutils and pciutils are already automatically included in the images when usbhost and pci are listed in MACHINE_FEATURES.

Reply:
Some of these packages are not available in poky/oe-core. In this case, you need properly document and instruct the user what are the extra layers needed to be included in the bblayers.conf.

Reply:
From where the brcmswitch package is coming from?

Reply:
brcmswitch package is still under testing , this package shouldn't been added in this commit and will be removed. usbutils ,pciutils and other duplicate packages will be removed